specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -111,6 +117,10 @@ enum NodeImageSelectionType {
 
   @doc("The image versions to upgrade nodes to are selected as described below: for each node pool in managed clusters affected by the update run, the system selects the latest image version such that it is available across all other node pools (in all other clusters) of the same image type. As a result, all node pools of the same image type will be upgraded to the same image version. For example, if the latest image version for image type 'AKSUbuntu-1804gen2containerd' is 'AKSUbuntu-1804gen2containerd-2021.10.12' for a node pool in cluster A in region X, and is 'AKSUbuntu-1804gen2containerd-2021.10.17' for a node pool in cluster B in region Y, the system will upgrade both node pools to image version 'AKSUbuntu-1804gen2containerd-2021.10.12'.")
   Consistent,
+
+  @added(Versions.v2024_05_02_preview)
+  @doc("Upgrade the nodes to the custom image versions. When set, update run will use node image versions provided in customNodeImageVersions to upgrade the nodes. If set, customNodeImageVersions must not be empty.")
```

Comment:
"When set, update run will use node image versions provided in customNodeImageVersions to upgrade the nodes; meanwhile, customNodeImageVersions must not be empty."

Reply:
original wording is clearer.